app/app.go

```diff
@@ -287,7 +288,7 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 
 	// Create IBC Keeper
 	app.IBCKeeper = ibckeeper.NewKeeper(
-		appCodec, keys[ibchost.StoreKey], app.StakingKeeper, scopedIBCKeeper,
+		appCodec, keys[ibchost.StoreKey], app.GetSubspace(ibchost.ModuleName), app.StakingKeeper, scopedIBCKeeper,
```

Comment:
is IBC needed actually for regen-ledger for mainnet?

Reply:
I don't see a reason for not having that. We can enable IBC transfers via governance when we feel comfortable.